QUARKUS-960.md

```diff
@@ -0,0 +1,29 @@
+# Quarkus-960 - Improved getting started experience with CLI
+JIRA link: https://issues.redhat.com/browse/QUARKUS-960
+
+There was an existing CLI tool in Quarkus that has been fully reworked as part of QUARKUS-960. 
+
+So, the goal is to verify proper functionality of the new Quarkus CLI tool.
+
+## Scope of the testing
+
+The Quarkus CLI is compatible with Maven, Gradle and jbang tools, however RHBQ is only focused on Maven, so Gradle and Jbang will be covered as part of the test coverage in upstream.
+
+Test development will focus on the supported features scoped for RHBQ 2.2:
+- Create applications (with a set of extensions)
+- Create extensions
+- Build existing Quarkus applications on JVM
+- Build existing Quarkus applications on Native (also using container configuration and builder images)
+- Run Quarkus applications on DEV mode
+- Run Quarkus applications on Remote Dev
+
```

Comment:
- Add tests for different BOMs usage as the BOM is gonna be split
 - Add tests for special characters in names and paths
 - Add tests for `help` for available commands and subcommands
 - Add test for command autocompletion (if possible
 - Add tests for properties management (list, search, add, remove, …)

Reply:
About autocompletion.. I'm ok to add this in the manual verification section, but not in the automated coverage as it would require some utils to be installed.

About the properties management, this is not planning to be added afaik

I'm ok to add the other three points.

Reply:
autocompletion ... keep it in the scope of the initial exploration
Maybe we could check that the file gets generated and covers some set of defined commands, wdyt?

Properties ... oki